.github/workflows/docker-build-and-push.yml

```
@@ -0,0 +1,415 @@
+name: Reusable Build and Push Docker Image
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)